src/data/shopping-list.ts

```diff
@@ -23,9 +27,22 @@ export const completeItem = (
   itemId: number,
   complete: boolean
 ): Promise<void> =>
-  hass.callApi("POST", "shopping_list/item/" + itemId, {
+  hass.callWS({
+    type: "shopping_list/items/update",
+    item_id: itemId,
     complete,
   });
 
 export const clearItems = (hass: HomeAssistant): Promise<void> =>
-  hass.callApi("POST", "shopping_list/clear_completed");
+  hass.callWS({
+    type: "shopping_list/items/clear",
+  });
+
+export const addItem = (
```

Comment:
dev seems to be missing my last PR for adding to the shopping list but I see it was merged. Not sure why that is?